components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.html

```diff
@@ -6,7 +6,7 @@ <h2 slot="title">Create {{ objectNoun | titlecase }}</h2>
           <chef-form-field>
             <label>
               <span class="label">Name <span aria-hidden="true">*</span></span>
-              <input chefInput formControlName="name" type="text" (keyup)="handleNameInput($event)" id="name-input">
+              <input chefInput formControlName="name" type="text" (keyup)="handleNameInput($event)" id="name-input" data-cy="create-name">
```

Comment:
this (and other `data-cy` additions) can cause problems in the acceptance cypress tests, since this attribute won't be available in acceptance until this changes gets promoted.